src/App.js

```diff
@@ -11,6 +11,7 @@ import { Login } from "./pages/Login/Login";
 import { SignUp } from "./pages/SignUp/SignUp";
 import { MyList } from "./pages/MyList/MyList";
 import { ToastContainer } from "react-toastify";
+import { History } from "./pages/History/History";
 
```

Comment:
you can use a common index.js file for all folder. It will reduce multiple lines of import